docs/detections/rules-ui-manage.asciidoc

```diff
@@ -139,6 +139,9 @@ Similarly, rules will be skipped if they can't be modified by a bulk edit. For e
 +
 NOTE: The action frequency you select applies to all actions (both new and existing) on all selected rules. If you don't want to change the frequency of existing actions, update the rules separately.
 
++
+IMPORTANT: In 8.8, action frequencies can be set for individual actions, and you can specify if notifications are sent when an alert is generated or at a custom alert summary interval. When you upgrade to 8.8, action-level frequency settings are applied to rules that have actions and were created in 8.7 or earlier. The frequency setting will default to the action frequency that was chosen for the rule before the upgrade. 
```

Comment:
I think we might need to rephrase this a little bit. Couple notes:
1. There will be three options in 8.8 in terms of notification frequency:
-  Notification of summary of alerts on rule run
-  Notification of summary of alerts on custom interval
-  Notification for each generated alert on rule run
2. Regarding the upgrade. After the upgrade, if user had `rule level notification frequency` (`"On each rule execution"`, `"Hourly"`, `"Daily"` or `"Weekly"`) set in 8.7, this `rule level notification frequency` will be converted into an action level frequency for each individual action in 8.8. The `"Perform no actions"` rule level frequency option will be indicated as an indefinitely snoozed rule.

<img width="382" alt="Screenshot 2023-05-11 at 14 06 47" src="https://github.com/elastic/security-docs/assets/2700761/c4c69664-ea06-41a4-8ced-0af7926fd4c3">

Reply:
@e40pud how important is it that we distinguish the two frequencies users can set when they select the alert as summary option? Would it be sufficient to lump the two options together and say that they can choose an alert summary interval? For example:

In 8.8, action frequencies can be set for individual actions, and you can choose to send notifications when an alert is generated or at a specified alert summary interval.

Reply:
Regarding the second half of the note, would the following work:

Action frequencies for rules created in 8.7 or earlier are converted from the rule-level to the action-level after you upgrade to 8.8. Scheduled actions will continue to run on their previously specified frequency (`On each rule execution` `Hourly`, `Daily`, or `Weekly`). Rules without actions will be snoozed indefinitely.